### DISCUSSION TITLE
[Feature] Add option to import shared album photos to personal library

### DISCUSSION THREAD

**Discussion by dwrtewd** [immich-app/immich#12184] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Currently, when users share photos through a common album in Immich, there's no direct way to add these shared photos to one's personal library. This limitation prevents shared photos from being included in search results, timeline views, and face recognition features.
The workaround I am using is manually downloading photos from the shared album, uploading them to their personal collection, and then removing duplicates.

Such a feature would enhance the usefulness of shared content by integrating it with personal libraries, especially if we can either import individual photots or a bulk selection.
A _nice to have_ option to go along with this feature would be auto-adding new photots from the album to the my library.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#1587
